ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bgp.py

```diff
@@ -21,6 +21,9 @@ def router_bgp(self) -> dict | None:
         """
         Return the structured config for router_bgp
         """
+        if self._overlay_cvx:
```

Comment:
```suggestion
        if self._overlay_cvx is True:
```

Reply:
why? We don't to that for these internal flags we produce our selves.

Reply:
The property we read is 
```py
    @cached_property
    def _overlay_cvx(self) -> bool:
        return get(self._hostvars, "switch.overlay.cvx") is True
```